packages/react-components/react-table/src/components/DataGridRow/useDataGridRow.tsx

```diff
@@ -52,6 +52,8 @@ export const useDataGridRow_unstable = (props: DataGridRowProps, ref: React.Ref<
 
   const onKeyDown = useEventCallback((e: React.KeyboardEvent<HTMLTableRowElement>) => {
     if (selectable && !isHeader && e.key === ' ') {
```

Comment:
I know what it might be an accessibility concern, but what if there will be an input in a row? It will be impossible to put anything into the input ([CodeSandbox](https://codesandbox.io/p/sandbox/eager-frog-0hwnqg?file=%2Fsrc%2FApp.tsx&selection=%5B%7B%22endColumn%22%3A15%2C%22endLineNumber%22%3A8%2C%22startColumn%22%3A15%2C%22startLineNumber%22%3A8%7D%5D)).

May be I don't understand the scenario, but if not - we will something like [that](https://github.com/microsoft/fluentui/blob/master/packages/fluentui/react-northstar/src/utils/shouldPreventDefaultOnKeyDown.ts).

Reply:
yeah it's a good point, the `shouldPreventKeydown` util is a bit too specific and is actually never used, so I opted for a utility that seemed a bit more useful 🤷